mqtt/mqtt-broker/src/configuration/size.rs

```diff
@@ -0,0 +1,249 @@
+use std::str::FromStr;
+
+use lazy_static::lazy_static;
+use regex::Regex;
+use serde::{
+    de::{Error as SerdeError, Visitor},
+    Deserialize, Deserializer,
+};
+
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub struct HumanSize(usize);
+
+impl HumanSize {
```

Comment:
nit: do you think we should name the file `human_size.rs`, by the name of the main struct in there?